server/gateway/config.json

```diff
@@ -50,14 +50,7 @@
     },
     "login": {
         "enabled": false,
-        "microsoft": {
-            "clientId": "",
-            "secret": ""
-        },
-        "linkedAccounts": {
-        },
```

Comment:
why?

Reply:
I'm guessing because you moved to env variables.

Reply:
Restored the empty config.json to have places for the release step to still fill in the secrets